libs/vkd3d/device.c

```diff
@@ -1029,7 +1029,7 @@ static HRESULT vkd3d_instance_init(struct vkd3d_instance *instance,
     application_info.engineVersion = vkd3d_get_vk_version();
     application_info.apiVersion = loader_version;
 
-    INFO("vkd3d-proton - build: %"PRIx64".\n", vkd3d_build);
+    INFO(vkd3d_build < (1ull << 60) ? "vkd3d-proton - build: %015"PRIx64".\n" : "vkd3d-proton - build: %016"PRIx64".\n", vkd3d_build);
```

Comment:
This feels kinda goofy. Why doesn't just a plain %016"PRIx64" work as is?

Reply:
Ye, sorry about that. It's because after my previous change in https://github.com/HansKristian-Work/vkd3d-proton/commit/61b13cc2786cdeeb96232f0ba7e2dd8aa789e012, `vkd3d_build` will be either `0xfffffffffffffff` on clean trees or `0xfffffffffffffff0` on dirty ones and afaict always printing with `%016` would prepend a misleading `0` to the former case.

Reply:
Right, but if the hash is legitimately less than 1ull << 60, this will be funny. I think we need to fix the meson build to mark dirtyness a different way then.